File: totalPrime1000.py
'''
    @author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
    Description of problem at: https://www.codeeval.com/open_challenges/4/
'''

total = 5
counter = 3
i = 4
while (counter < 1001):
    if ((i % 2) == 0): 
        i+=1
        continue
    j = 3
    while j <= i:
        if ((i % j == 0) and (i != j)): 
            break
        if (j == i):
            total+=i
            counter+=1
        j+=2
    i+=1

print (total)

File: 04/totalPrime1000.php
/*
 *  @author Sarah Kidd (www.linkedin.com/in/sarahkidd/)
 *  Description of problem at: https://www.codeeval.com/open_challenges/4/
 */

<?php
	$total = 5;
	$counter = 3;
	for ($i = 4; $counter < 1001; $i++) {
		if ($i % 2 == 0)
			continue;
		for ($j = 3; $j <= $i; $j+=2) {
			if ($i % $j == 0 && $i != $j)
				break;
			if ($j == $i) {
				$total+=$i;
				$counter++;
			}
		}
	}
	echo "$total";
?>
